certdocs/TRAQ-138-SDD.md

```diff
@@ -920,6 +926,21 @@ positive and negative filtering criteria for the files within those folders.
 - Verification: Test
 - Safety Impact: None
 
+### completion_cmd.go
```

Comment:
seeing as you've added this section, I suppose you should add sections for the other _cmd.go files to be consistent, and move the associated requirements there.

also, (nit) REQ-TRAQ-SWL-17 should  be moved from main.go to req.go section.